app.js

```diff
@@ -57,6 +66,44 @@ var server = app.listen(port, function () {
     console.log('Server running at http://127.0.0.1:' + port + '/');
 });
 
+//initial meeting endpoints
+
+//create a meeting
+app.post("/meeting", function (req, res) {
+    var start_datetime = req.query.start_datetime;
+    var end_datetime = req.query.end_datetime;
+    var room_number = req.query.room_number;
+
+    console.log(req);
+
+    var sql = "INSERT INTO ebdb.Meeting (start_datetime, end_datetime, room_number) VALUES (?, ?, ?);"
+    var inserts = [start_datetime, end_datetime, room_number];
+    mysql.format(sql, inserts);
+
+    console.log(mysql.format(sql, inserts));
+
+    pool.getConnection(function(err,connection) {
+        connection.query(sql, inserts, function(error, results, fields) {
```

Comment:
We should be auto generating a meeting_id (primary key) when we create a meeting in this table.  The database should be able to handle this by setting ``AUTO_INCREMENT`` on the field in the table definition.  After successfully inserting the meeting into the table we should determine what the automatically generated meeting_id is so we can pass it back to the client in the response to the POST.

The response from a successful POST command should have a HTTP status of 201 ("created") and contain a Link header that points to the URL of the generated resource (will conform to /meeting/{meeting_id} pattern) .

Reply:
Also, per the original ticket, we should add a ``createdAt`` field in the DB when the meeting is created, this can be done by the DB automatically by defining the field in the table as

```sql
createdAt TIMESTAMP NOT NULL DEFAULT NOW()
```
and even though it wasn't in the original ticket, I think it would make sense to add a column the last time the record was updated

```sql
updatedAt TIMESTAMP NOT NULL DEFAULT NOW() ON UPDATE NOW()
```

Reply:
Sample table created in the DB has an AUTO_INCREMENT set for id. Also added createdAt and updatedAt columns